build.sbt

```diff
@@ -105,9 +105,6 @@ pomExtra := (
   </scm>
 )
 
-// Temporary disable publishing of doc in dotty, can't get it to build.
-publishArtifact in (Compile, packageDoc) := !scalaBinaryVersion.value.startsWith("3")
```

Comment:
@SethTisue With this removed, I still get this error when trying to publish for Scala 3: 

```
[info] Main Scala API documentation successful.
[error] java.io.FileNotFoundException: /home/cheeseng/git/scalatestplus/scalatestplus-testng/target/scala-3.0.1/api/lib/template.css (No such file or directory)
[error] 	at java.io.FileInputStream.open0(Native Method)
[error] 	at java.io.FileInputStream.open(FileInputStream.java:195)
[error] 	at java.io.FileInputStream.<init>(FileInputStream.java:138)
[error] 	at scala.io.Source$.fromFile(Source.scala:94)
[error] 	at scala.io.Source$.fromFile(Source.scala:79)
[error] 	at $7661312474db26131063$.docTask(build.sbt:114)
[error] 	at $67886ffeb36791a2339d$.$anonfun$$sbtdef$1(/home/cheeseng/git/scalatestplus/scalatestplus-testng/build.sbt:137)
[error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
[error] 	at sbt.std.Transform$$anon$4.work(Transform.scala:68)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:282)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:23)
[error] 	at sbt.Execute.work(Execute.scala:291)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:282)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:265)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:64)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[error] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[error] 	at java.lang.Thread.run(Thread.java:748)
[error] (Compile / doc) java.io.FileNotFoundException: /home/cheeseng/git/scalatestplus/scalatestplus-testng/target/scala-3.0.1/api/lib/template.css (No such file or directory)
[error] Total time: 4 s, completed Sep 10, 2021 6:45:55 PM
```

I'll try to add back this line first for now, will see if there's better way to fix it in future.

Reply:
weird! haven't seen that one